backend/spec/factories/surveyors.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  factory :surveyor do
+    user { nil }
+    firstname { "MyString" }
+    lastname { "MyString" }
+    email { "MyString" }
+    phone { "MyString" }
+    street_address { "MyString" }
+    geocode { "MyString" }
+    city { "MyString" }
+    zipcode { "MyString" }
+    state { "MyString" }
+    role { "MyString" }
+    status { "MyString" }
```

Comment:
@aTegart Usually before committing this I'll update the sample values here from `MyString` to something that looks like the data we'll be using. You can look at mine for an example.